input/pagecontent/reference.md

```diff
@@ -654,6 +660,18 @@ A special case of the ElementDefinition path is setting properties of the first
   . ^short
   ```
 
+* The path to the designation value of a concept:
+
+  ```
+  #SomeCode ^designation[0].value
+  ```
+
+* The path to the property code of a concept within a hierarchy:
+
+  ```
+  #SomeCode #ContainedCode ^property[0].code
```

Comment:
I'd suggest `#ChildCode` rather than `#ContainedCode` -- as this relationship does not always indicate _containment_ (but the FHIR spec _does_ refer to it as a "child concept").
```suggestion
  #SomeCode #ChildCode ^property[0].code
```